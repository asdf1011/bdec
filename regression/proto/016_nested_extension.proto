message Original {
  required int32 a = 1;
  extensions 100 to max;
}

message Other {
    extend Original {
      required int32 b = 100;
    }

    extend Original {
      required int32 c = 101;
    }
    required Original d = 1;
}
