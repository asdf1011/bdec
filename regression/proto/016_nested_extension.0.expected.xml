<Other>
    <d>
        <a>9</a>
        <b>99</b>
        <c>999</c>
    </d>
</Other>
